server/src/app.js

```diff
@@ -19,10 +20,10 @@ connect_db().catch((err) => {
   // just logging, not trying to recover here. DB connection is reattempted per-request below
 });
 
-const schema = create_schema();
 const app = express();
 
-app.use(body_parser.json({ limit: "50mb" }));
+//TODO: figure out if we need this anymore, since apollo seems to work without it?
+// app.use(body_parser.json({ limit: "50mb" }));
```

Comment:
Hmm, don't even remember what this was for. Maybe introspection queries got too big for the express defaults? Probably should have only been on in dev if that was the case, oops.

Reply:
nice find, got rid of it